app/controllers/artists_controller.rb

```diff
@@ -44,7 +62,7 @@ def destroy
   private
 
   def artist_params
-    params.require(:artist).permit(:name, :description, :image_url, :instructor, :dj)
+    params.require(:artist).permit(:name, :description, :image_url, :instructor, :dj, :first_video_id, :second_video_id, :third_video_id, dance_style_ids: [], event_ids: [])
```

Comment:
I think you'll probably want to convert the videos param into one videos array. Probably best to make this fix so we don't have to change the database structure later. @ogarocious

Reply:
Hm. looks like you still have these 3 different video ids as required? I thought you had added a videos array?